### PR TITLE
Update base image exceptions list

### PR DESCRIPTION
**Pull request by justaugustus**
#### What type of PR is this:

/kind cleanup documentation

#### What this PR does / why we need it:

- Fixup table formatting for exceptions list
- Split exceptions into release and non-release images
- Update exception images and assumed OWNERS
- Update exceptions for release images

ref: https://github.com/kubernetes/kubernetes/issues/70249, https://github.com/kubernetes/kubernetes/issues/90698, https://github.com/kubernetes/kubernetes/pull/90804, https://github.com/kubernetes/kubernetes/pull/90674

### CODE REVIEW

**Comment by justaugustus** [comment]
/assign @tpepper @dims 
/area dependency security
cc: @kubernetes/release-engineering

File: release-engineering/baseimage-exception-list.md [hunk 1]
```diff
@@ -4,22 +4,69 @@ Theoretically, we expect to have all kubernetes images based on distroless (Plea
...
-|kube-controller-manager | k8s.gcr.io/debian-base| Causes performance regression, requires further investigation on klog when running 2000 nodes scalability test. OWNER:@yuwenma|
-|kube-scheduler | k8s.gcr.io/debian-base| Causes performance regression, requires further investigation on klog when running 2000 nodes scalability test. OWNER:@yuwenma|
+## Release images
+
+### Release: `debian-base`
+
+| Image Name | Reasons for exception |
+|---|---|
+| [debian-iptables] | Needed to supported images that require `iptables` |
+| [kube-controller-manager][core-images] | `klog` 2000 node scalability regressions, [Flexvolume master driver is blocking controller-manager from moving to distroless](https://github.com/kubernetes/kubernetes/issues/78737) |
```
Comment (dims) [inline release-engineering/baseimage-exception-list.md:21]: Please remove the `"klog" 2000 node scalability regressions,` , no longer applicable

Comment (justaugustus) [reply release-engineering/baseimage-exception-list.md]: @dims -- Fixed!

**Comment by tpepper** [comment]
lgtm but will leave for the other Tim @tallclair

**Comment by tallclair** [comment]
/lgtm

thanks!

**Pull request merged**